src/EscapeAnalysis.jl

```diff
@@ -137,18 +137,18 @@ end
 # ========
 
 """
-    struct EscapeLattice
+    x::EscapeLattice
 
 A lattice for escape information, which holds the following properties:
-- `Analyzed`: not formally part of the lattice, indicates this statement has not been analyzed at all
-- `ReturnEscape`: indicates it will escape to the caller via return (possibly as a field)
-- `ArgEscape`: indicates it will escape to the caller through `setfield!` on argument(s)
-  -1 : no escape
-   0 : unknown or multiple
-   n : through argument N
+- `x.Analyzed`: not formally part of the lattice, indicates this statement has not been analyzed at all
+- `x.ReturnEscape`: indicates it will escape to the caller via return (possibly as a field)
+- `x.ArgEscape`: indicates it will escape to the caller through `setfield!` on argument(s)
+  * `-1` : no escape
+  * `0` : unknown or multiple
+  * `n` : through argument N
 
 These attributes can be combined to create a partial lattice:
-(note that this is inverted from the order used for the lattice in Core.Compiler)
+(note that this is inverted from the order used for the lattice in `Core.Compiler`)
```

Comment:
This is no longer inverted now?

Reply:
I think it's still "reverted", because `find_escapes` transitions `LatticeElement` from top to bottom (where as inference does from bottom to top) ?

Reply:
Hm, my reply was kinda nonsense.

The lattice ordering is sort of orthogonal to which direction we transition lattice elements.

I will remove this sentence, and add another note on the contrast with `Core.Compiler` on their lattice transition